apps/openmw/mwbase/mechanicsmanager.hpp

```diff
@@ -90,6 +90,10 @@ namespace MWBase
             virtual void setPlayerClass (const ESM::Class& class_) = 0;
             ///< Set player class to custom class.
 
+            virtual void restoreDynamicStats(MWWorld::Ptr actor, bool sleep) = 0;
+            ///< Restores actor's dynamic stats during player's rest
```

Comment:
We don't use this kind of comment anymore (///<, comment after function), because several developers complained that it screws with functions in their editor/IDE.